cartographer_ros/cartographer_ros/node.cc

```diff
@@ -459,7 +459,10 @@ void Node::FinishAllTrajectories() {
 
 void Node::FinishTrajectory(const int trajectory_id) {
   carto::common::MutexLocker lock(&mutex_);
+  CHECK(is_active_trajectory_.count(trajectory_id) > 0 &&
```

Comment:
Yes, that's what I meant. The `CHECK` will fail in this case, so it is irrelevant whether we added a new value to the map.